packages/frontend-2/components/settings/workspaces/Billing.vue

```diff
@@ -2,34 +2,16 @@
   <section>
     <div class="md:max-w-xl md:mx-auto pb-6 md:pb-0">
       <SettingsSectionHeader title="Billing" text="Your workspace billing details" />
-      <CommonCard v-if="versionCount" class="text-body-xs bg-foundation">
+      <CommonCard class="text-body-xs bg-foundation">
         <p class="text-foreground font-medium">Workspaces are free while in beta.</p>
-        <p class="py-6">
-          Once the beta period ends, workspaces are still free up to
-          {{ versionCount.max }} model versions.
-          <br />
-          To store more versions across your projects you will need to upgrade to a paid
-          plan.
+        <p class="pt-1">
+          Once the beta period ends, your workspace will be converted to a free 30-day
```

Comment:
@benjaminvo I had this stashed from last night, I just updated the copy based on what we discussed this morning, but please suggest if you have anything better 😁

Reply:
@Mikehrn I think it's pretty good! Minor adjustments:

"After the beta ends, your workspace will start a 30-day free trial. You'll have full access until either the trial expires or you upgrade to a paid plan."